express.js

```diff
@@ -0,0 +1,14 @@
+const extensions = require('./index');
+
+const express = extensions.express = extensions.express || {};
```

Comment:
Why do this if we created this file explicitly to not include server-side code in the root entrypoint?

Reply:
because it's safer?

Reply:
I would not attach changes directly to either `extensions` or `extensions.express` if that is the goal.

If some part of our code requires the `express` entrypoint, the root entrypoint will magically also gain all of the `express` methods because that's the same object reference. Maybe do a clone here somewhere?